### PR TITLE
GCB-183: Add basic Travis integration

### PR DESCRIPTION
**Pull request by Augustyniak**
I decided to use `xcodebuild` as `fastlane` currently [doesn't support tvos](https://github.com/fastlane/fastlane/issues/3346).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: objective-c
```
Comment (ochococo) [inline .travis.yml]: I'm no expert but this seems not right.

**Comment by ochococo** [comment]
Only one comment, rest looks good. @Augustyniak 🎾

**Comment by Augustyniak** [comment]
I agree that it looks weird but AFAIK it is ok - according to my knowledge there you can't specify `swift` language. Other swift projects seems to agree on that - [example from Alamofire](https://github.com/Alamofire/Alamofire/blob/master/.travis.yml).

**Comment by Augustyniak** [comment]
Answered. 🎾

**Comment by ochococo** [comment]
OK, merge 💣

**Pull request merged**